pkg/cosign/verify.go

```diff
@@ -484,6 +484,12 @@ func VerifyImageSignatures(ctx context.Context, signedImgRef name.Reference, co
 	// entity that minimizes registry requests when supplied with a digest input
 	digest, err := ociremote.ResolveDigest(signedImgRef, co.RegistryClientOpts...)
 	if err != nil {
+		if strings.Contains(err.Error(), "MANIFEST_UNKNOWN") {
+			return nil, false, &VerificationError{
+				errorType: ErrImageTagNotFoundType,
+				message:   fmt.Sprintf("%s: %s", ErrImageTagNotFoundMessage, err),
```

Comment:
Please, use `%s: %w`

Reply:
cc @ChrisJBurns

Reply:
Yep, will try and commit the fix tonight

Reply:
Awesome! Thanks @ChrisJBurns

Reply:
@hectorj2f  I get the following come back when I change to `%w` : `fmt.Sprintf does not support error-wrapping directive %w`

Reply:
true! please, use %v instead if you use `err`.